Popularity/Kipras/pyspark/rollOnSpark.sh

```diff
@@ -0,0 +1,249 @@
+#!/bin/bash
```

Comment:
Kipras, that's good, I haven't had a chance to go into details since my
internet connection is quite poor in a place where I'm right now. Let
discuss things next week, I'm traveling home this week-end.
Best,
Valentin.

On Fri, Jul 29, 2016 at 10:14 AM, Kipras <notifications@github.com> wrote:

> In Popularity/Kipras/pyspark/rollOnSpark.sh
> <https://github.com/dmwm/DMWMAnalytics/pull/28#discussion_r72797994>:
>
> > @@ -0,0 +1,249 @@
> > +#!/bin/bash
>
> But these scripts are doing exactly that.
>
>    - rollOnSpark.sh does the local data transform and starts run.sh
>    - run.sh moves data to HDFS and starts SPARK job (pyspark_ml.py)
>    - pyspark_ml.py do ML on SPARK and return output
>    - run.sh transform output
>    - rollOnSpark.sh measure accuracy and plot result
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/dmwm/DMWMAnalytics/pull/28/files/bca227570064b0ea8e823e4ff9ae497274ced250#r72797994>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAHo0o787bkYo3LY7KB4Yc8DBUBkfl03ks5qagrXgaJpZM4JXFlB>
> .
>